http/client.go

```diff
@@ -427,6 +427,44 @@ func (c *InternalClient) marshalImportPayload(index, field string, shard uint64,
 	return buf, nil
 }
 
+// Forward the request to the new host in request
+func (c *InternalClient) Forward(ctx context.Context, w http.ResponseWriter, req *http.Request, scheme, host string) error {
+
+	body, err := ioutil.ReadAll(req.Body) // it appears I must consume the
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return nil
+	}
+
+	req.Body = ioutil.NopCloser(bytes.NewReader(body))
+	url := fmt.Sprintf("%s://%s%s", scheme, host, req.RequestURI)
+
+	forReq, err := http.NewRequest(req.Method, url, bytes.NewReader(body))
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return nil
+	}
+
+	forReq.Header = make(http.Header)
+	for h, val := range req.Header {
+		forReq.Header[h] = val
+	}
+
+	resp, err := c.httpClient.Do(forReq.WithContext(ctx))
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadGateway)
+		return nil
+	}
+	if resp.StatusCode != http.StatusOK {
+		http.Error(w, "Forward Failed", resp.StatusCode)
```

Comment:
I think we should maintain the same body from the original response instead of replacing with "Forward Failed". Maybe we could prepend "forward failed: " in the spirit of Go error messages adding context to the error.